book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -318,4 +318,4 @@ Conflicts:
 #
 ----
 
-You can modify that message with details about how you resolved the merge if you think it would be helpful to others looking at this merge in the future – why you did what you did, if it's not obvious.
+If you think it would be helpful to others looking at this merge in the future, you can modify this commit message with details about how you resolved the merge and explain why you did the changes you made if these are not obvious.
```

Comment:
This is still quite a mouthful. What about this?

> If you think it would be helpful to others looking at this merge in the future, you can modify this commit message with details about how you resolved the merge, and explain why you made changes if it's not obvious.